src/slack/publishMessages.js

```diff
@@ -25,11 +32,11 @@ module.exports = async ({ channel, repository, branches }) => {
 
   const blocks = composeBlocks({ repository, branches });
 
-  // eslint-disable-next-line no-plusplus
-  for (let i = 0; i < blocks.length; i++) {
-    // eslint-disable-next-line no-await-in-loop
-    await postMessage({ channel, blocks: [blocks[i]] });
-  }
+  blocks.forEach((block) => postMessage({ channel, blocks: [block] }));
 
   return response;
 };
+
+module.exports = defaultFunction;
+module.exports.normalHeaderMessage = normalHeaderMessage;
+module.exports.zeroStaleHeaderMessage = zeroStaleHeaderMessage;
```

Comment:
I guess it's not necessary to expose these functions?

Reply:
it's so that the tests can access them without having to hardcode the messages (https://github.com/prodigyeducation/branch-cleaner/pull/46/files#diff-4f69016cba43f83eb449de28efb5c9ccR42) Or maybe there should be a `messages.js` only for message definitions?